retrofit/src/main/java/retrofit2/Platform.java

```diff
@@ -32,12 +32,14 @@ static Platform get() {
   }
 
   private static Platform findPlatform() {
-    try {
-      Class.forName("android.os.Build");
-      if (Build.VERSION.SDK_INT != 0) {
-        return new Android();
+    if ("Dalvik".equals(System.getProperty("java.vm.name"))) {
```

Comment:
BTW you can replace the forName check for Java 8 by using the `java.class.version` property. Any value >= 52 is Java 8 or newer.